gulpfile.js

```diff
@@ -7,7 +7,7 @@ var gulp = require('gulp'),
   fs = require('fs-extra'),
   runSequence = require('run-sequence'),
   inlineResources = require('./tools/gulp/inline-resources');
-
+  require('@angular/compiler-cli');
```

Comment:
This is necessary because gulp fails when running `npm run build` without it. It specifies some config with ngc I think. You can remove this and run `npm run build` to see what I mean.